Sources/PackagePlugin/PackageManagerProxy.swift

```diff
@@ -62,6 +62,12 @@ public struct PackageManager {
         /// Whether to print build logs to the console
         public var echoLogs: Bool
 
+        /// Whether to print build progress to the console
+        ///
+        /// If this is set to true, `BuildResult.logText` will not contain any
+        /// progress information.
+        public var progressToConsole: Bool
```

Comment:
This is new public API and will therefore need a Swift Evolution proposal API availability annotation.

Can you please start a pitch on the Swift Forums to propose this new API?

Reply:
Sure, I started it here: https://forums.swift.org/t/pitch-build-progress-display-control-for-swiftpm-plugins/78678